vsintegration/tests/UnitTests/InlineParameterNameHintTests.fs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
```

Comment:
What about constructors? Default ones, custom ones, and for multi-cased DUs?

Reply:
So hints for those are not present yet. I will add them in follow-ups - for now, I added test cases to keep track.